src/table/table.stories.ts

```diff
@@ -472,20 +474,28 @@ storiesOf("Table", module).addDecorator(
 	.addDecorator(withKnobs)
 	.add("default", () => ({
 		template: `
-		<div style="width: 650px">
-			<app-table
-				[model]="model"
-				[size]="size"
-				[showSelectionColumn]="showSelectionColumn"
-				[striped]="striped"
-				[sortable]="sortable"
-				[isDataGrid]="isDataGrid">
-			</app-table>
-		</div>
+		<ibm-table-container>
+			<ibm-table-header>
+				<h4 ibmTableHeaderTitle>{{title}}</h4>
+				<p ibmTableHeaderDescription>{{description}}</p>
+			</ibm-table-header>
+			<div style="width: 650px">
```

Comment:
Is the `div` necessary? Why not apply styles to table directly?

Reply:
Ya it doesn't work otherwise

Reply:
Why doesn't it work? What does it do instead?

Reply:
Well since its only two columns so the table shrinks down

Reply:
What if you add width to the table? Might need to also add a display value with it.

Reply:
Just confirming that the `width: 650px` is needed because of the centering styles in the storybook?